cli/args/mod.rs

```diff
@@ -1817,22 +1367,31 @@ fn try_resolve_node_binary_main_entrypoint(
   }
 }
 
+#[derive(Debug, Error)]
+#[error("Bad URL for import map.")]
+pub struct ImportMapSpecifierResolveError {
+  #[source]
+  source: deno_path_util::ResolveUrlOrPathError,
+}
```

Comment:
I shifted this code out of the CLI, but then I moved it back. I kept the changes.